config/packages/test/sylius_uploader.yaml

```diff
@@ -5,7 +5,9 @@ services:
             - "%sylius.uploader.filesystem%"
         factory: ['@knp_gaufrette.filesystem_map', 'get']
     sylius.image_uploader:
-        class: Sylius\Behat\Service\Uploader\ImageUploader
+        class: Sylius\Component\Core\Uploader\ImageUploader
         arguments:
             - "@sylius_test.filesystem"
-        public: true
+            - "@Sylius\\Component\\Core\\Generator\\ImagePathGeneratorInterface"
+    Sylius\Component\Core\Generator\ImagePathGeneratorInterface:
```

Comment:
I guess after the recent changes we just need this service to make tests work and the rest of definitions in this file can be removed.